apps/src/templates/teacherHomepage/CollapsibleSection.jsx

```diff
@@ -58,7 +58,8 @@ const CollapsibleSection = React.createClass({
     header: React.PropTypes.string.isRequired,
     linkText: React.PropTypes.string,
     link: React.PropTypes.string,
-    collapsible: React.PropTypes.bool
+    collapsible: React.PropTypes.bool,
+    showLink: React.PropTypes.bool
```

Comment:
would it make sense to just not pass in a link prop in the case where we dont want to show a link?